tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -498,6 +498,15 @@ function () {
 			}
 		);
 
+		remove_all_filters( 'webp_uploads_image_sizes_with_additional_mime_type_support' );
```

Comment:
I don't think these lines are needed.

The test suite resets all hooks during tear down so all previous filters are already gone.

Reply:
Sounds good. Updated.